Daily reward show split button and multiple wallet

pantelisss opened the pull request:
## **Why?**
Show reward splits in reward details
### **How?**
- Added reward split domain model
- Show split button if reward split contains more than one wallets
- Implemented bottom sheet to show the wallets percentage
### **Testing**
The following stations have reward splits: Agreeable Emerald Solar, Immense Nylon Altocumulus, Damp Azure Scud, Exotic Pecan Aurora
Make sure the splits are rendered properly and the split button is hidden when there are no splits
### **Additional context**
fixes fe-1075

PavlosTze reviewed (changes requested):
All good except three (minor) UI issues:
1. I think we need a bit more top padding in the Bottom Sheet between the drag handler
2. I think we need a bit more padding above the "Reward Split" title in the bottom sheet
3. In the Daily Reward screen, for the station `High Denim Ice` in emulator iPhone 15, the last character in the $WXM is truncated and replaced with `...`, I remember in a call saying that we should use autosized text there to avoid truncations, in Android I used `minTextSize = 24sp , maxTextSize = 32sp`

pantelisss commented:
> All good except three (minor) UI issues:
> 
> 1. I think we need a bit more top padding in the Bottom Sheet between the drag handler
Unfortunately this is grabber is added by the system
> 2. I think we need a bit more padding above the "Reward Split" title in the bottom sheet
Fixed
> 3. In the Daily Reward screen, for the station `High Denim Ice` in emulator iPhone 15, the last character in the $WXM is truncated and replaced with `...`, I remember in a call saying that we should use autosized text there to avoid truncations, in Android I used `minTextSize = 24sp , maxTextSize = 32sp`
fixed


Ready to check again
